SECURTITY.md

```diff
@@ -1,41 +0,0 @@
-<!-- BEGIN MICROSOFT SECURITY.MD V0.0.7 BLOCK -->
```

Comment:
Note to self/reviewers. File is removed due to duplicate present on target branch. Naming is also incorrect.